Server/Server.Exec/MainServer.cpp

```diff
@@ -10,13 +9,14 @@ using Server::Builder::ServerBuilder;
 int main(int argc, const char** argv)
 {
     ArgParser parser(argc, argv);
-    auto      server = ServerBuilder()
-                                      .setValue(parser.getPair<std::string>("--port"))
-                                      .setValue(parser.getPair<std::string>("--hostaddr"))
-                                      .setValue(parser.getPair<std::string>("--dbname"))
-                                      .setValue(parser.getPair<std::string>("--user"))
-                                      .setValue(parser.getPair<std::string>("--password"))
-                                      .make();
+    auto server = ServerBuilder()
+                                 .setValue(parser.getPair("--port"))
+                                 .setValue(parser.getPair("--dbport"))
+                                 .setValue(parser.getPair("--hostaddr"))
+                                 .setValue(parser.getPair("--dbname"))
+                                 .setValue(parser.getPair("--user"))
+                                 .setValue(parser.getPair("--password"))
+                                 .make();
 
     server->start();
 
```

Comment:
Please add a variable instead of true in the while loop, this will eliminate the magic number problem

Reply:
Actually no. This code doesn't violate clean code principles. 
Don't look for mistakes out of the blue